include/varargs.h

```diff
@@ -0,0 +1,220 @@
+/* Record that this is varargs.h; this turns off stdarg.h.  */
```

Comment:
Ditto about licenses. Possibly worth moving these 3 files to a subdirectory of `include`, similar to `include/PR/`.